src/components/ProjectSettingsTabs/GithubCommitQueueTab/GithubTriggerAliasField.tsx

```diff
@@ -0,0 +1,71 @@
+import styled from "@emotion/styled";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Body, InlineCode } from "@leafygreen-ui/typography";
+import { Field } from "@rjsf/core";
+import { size } from "constants/tokens";
+import { PatchTriggerAliasStatus } from "../utils";
+
+export const GithubTriggerAliasField: Field = ({ formData }) => {
+  const {
+    alias,
+    childProjectIdentifier,
+    parentAsModule,
+    status,
+    taskSpecifiers,
+  } = formData;
+  return (
+    <Tooltip
+      align="right"
+      trigger={<TooltipTrigger data-cy="pta-item">{alias}</TooltipTrigger>}
+      data-cy="pta-tooltip"
+    >
+      <Body>
+        <strong>Project:</strong> {childProjectIdentifier}
+      </Body>
+      {parentAsModule && (
+        <Body>
+          <strong>Module:</strong> {parentAsModule}
+        </Body>
+      )}
+      {status && (
+        <Body>
+          <strong>Wait On:</strong> {PatchTriggerAliasStatus[status]}
+        </Body>
+      )}
+      {!!taskSpecifiers.length && (
+        <>
+          <Body>
+            <strong>Variant/Task Regex Pairs</strong>
```

Comment:
I noticed that these don't really work on smaller screens. Doing this in a separate ticket works as well. 
<img width="433" alt="image" src="https://user-images.githubusercontent.com/13104717/159063623-f31ec246-2455-4531-9871-4aaa525b6e73.png">

Reply:
nice! thanks!